apiserver/pkg/printers/projectcalico/printers.go

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2020-2021 Tigera, Inc. All rights reserved.
+// Copyright 2017 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package projectcalico
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/duration"
+	"k8s.io/kubernetes/pkg/printers"
+
+	calico "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
+)
+
+// AddHandlers adds print handlers for projectcalico types.
+func AddHandlers(h printers.PrintHandler) {
+	calicoNodeStatusColumnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "Node", Type: "string", Priority: 0, Description: "The node name identifies the Calico node instance for node status."},
+		{Name: "Classes", Type: "string", Description: "Types of information to monitor for this calico/node."},
+		{Name: "Update Interval", Type: "string", Description: "Period in seconds at which CalicoNodeStatus should be updated."},
+		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
+		{Name: "last Updated", Type: "string", Description: "Last time the status has been updated"},
+		{Name: "Agent State", Type: "string", Priority: 1, Description: "The state of the BGP Daemon."},
+		{Name: "BGP Peers", Type: "string", Priority: 1, Description: "Number of BGP Peers in the format of total/established/non-established."},
```

Comment:
I think a nicer format might just be "established / total" - e.g., "5/5" means all are established, "3/5" means two are not.